packages/twenty-server/src/engine/core-modules/environment/environment-variables.ts

```diff
@@ -375,12 +375,12 @@ export class EnvironmentVariables {
       '"WORKSPACE_INACTIVE_DAYS_BEFORE_NOTIFICATION" should be strictly lower that "WORKSPACE_INACTIVE_DAYS_BEFORE_DELETION"',
   })
   @ValidateIf((env) => env.WORKSPACE_INACTIVE_DAYS_BEFORE_DELETION > 0)
-  WORKSPACE_INACTIVE_DAYS_BEFORE_NOTIFICATION = 30;
+  WORKSPACE_INACTIVE_DAYS_BEFORE_NOTIFICATION = 12;
```

Comment:
2 days difference seems a bit ambitious. What happens if the cron is broken for 1 or 2 days, then we will delete the workspace without warning

Reply:
Yes let's default to 7 and 14. On day 0, I hope Stripe sends something... Not sure but I'd be surprised if it's not possiible